documentation/doc-Migration_Toolkit_for_Virtualization/master.adoc

```diff
@@ -59,6 +59,8 @@ include::modules/compatibility-guidelines.adoc[leveloffset=+2]
 
 You can install the {operator-name} by using the {ocp} web console or the command line interface (CLI).
 
+In {project-first} version 4.2 and later, the {operator-name} includes the {project-short} web console.
```

Comment:
you mean 2.4, right? :)

Reply:
yes.

Reply:
@ahadas This is just the start of my work. I've changed it to a draft PR. Sorry for the inconvenience.